WP_Serve_File.php

```diff
@@ -89,7 +100,7 @@ public function serve_file($name = '') {
     header('Pragma: cache');
     header('Content-Type: ' . $contentType);
     echo $content;
-    die();
+    wp_die();
```

Comment:
This is an unrelated change. IIRC I used `die` on purpose here, but I'm not sure. Either way, it shouldn't be in this PR.